lib/users/routes.js

```diff
@@ -30,18 +35,203 @@ const all = (req, res, next) => {
 
 /**
  * Add a new User
+ * @param {object} req - HTTP Request
+ * @param {object} res - HTTP Response
+ * @param {object} next - Express callback
+ *
+ * @returns {promise} content-types promise
  */
-const add = () => {};
+const add = (req, res, next) => {
+  const errors = _.get(req.session, 'form.users.save.errors', {});
+  const values = _.get(req.session, 'form.users.save.content', {});
+  const merged = req.app.get('users-model') || {};
```

Comment:
This is actually a mistake I made when writing the story. In the current implementation, password is not required in either situation. Adding a required password _should_ be out of scope for this story. Reasoning:

* Passwords are not currently required
* User forms are created using `content-types` and we would have to do some serious manipulation of the `merged` content-type object to make this have a required password in one situation and not have passwords required in another

I think that in the [first time setup story](https://github.com/punchcard-cms/punchcard/issues/317) the solution will be to have a different form for first time users that requires a password. This form can then _also_ be used when adding a new user.

request: I can skip your `required` password comments in this PR and defer to the first-time user story to add that functionality as that is _new_ and should not be part of this refactor